internal/pkg/config/config.go

```diff
@@ -127,6 +128,34 @@ func newDefaultWalletConfig() *WalletConfig {
 	}
 }
 
+// DrandConfig holds all configuration options related to pulling randomness from Drand servers
+type DrandConfig struct {
+	// Addresses are are drand server addresses in the format
+	Addresses []string `json:"addresses"`
+	// Secure is whether or not the drand address are secure (e.g. TLS)
+	Secure bool `json:"secure"`
+	// DistKey is the distributed public key of the server group expressed as hex encoded coefficients
+	DistKey []string `json:"distKey"`
+}
+
+func newDefaultDrandConfig() *DrandConfig {
+	return &DrandConfig{
+		Addresses: []string{
+			"localhost:8080",
```

Comment:
Eventually the public network's addresses go here right?  When will we switch over?

Reply:
When we know what addresses we'll be using.